cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraScheduler.java

```diff
@@ -88,6 +93,9 @@ public void resourceOffers(final SchedulerDriver driver, final List<Offer> offer
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("> resourceOffers(driver : {}, offers : {})", driver, protoToString(offers));
         }
+
+        configuration.getSeedManager().syncSeeds(clock.now().getMillis());
```

Comment:
There is rationale in moving syncing seeds to another thread in case then http connection blocks.
Fixed. Updated SeedManager to use ScheduledExecutorService.